operators/admin/olm-managing-po.adoc

```diff
@@ -0,0 +1,47 @@
+:_content-type: ASSEMBLY
+[id="olm-managing-po"]
+= Managing platform Operators (Technology Preview)
+include::_attributes/common-attributes.adoc[]
+:context: olm-managing-po
+
+toc::[]
+
+A platform Operator is an OLM-based Operator that can be installed during or after an OpenShift Container Platform cluster's Day 0 operations and participates in the cluster's lifecycle. As a cluster administrator, you can manage platform Operators by using the `PlatformOperator` API.
+
+:FeatureName: The platform Operator type
+include::snippets/technology-preview.adoc[]
+
+include::modules/arch-platform-operators.adoc[leveloffset=+1]
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../operators/understanding/olm-packaging-format.adoc#olm-rukpak-about_olm-packaging-format[RukPak component and packaging format]
+
+include::modules/olm-po-techpreview.adoc[leveloffset=+2]
+
+[id="prerequisites_olm-managing-po"]
+== Prerequisites
+
+- Access to an {product-title} cluster using an account with `cluster-admin` permissions.
+- The `TechPreviewNoUpgrades` feature set enabled on the cluster.
+- Only the `redhat-operators` catalog source enabled on the cluster. This is a restriction during the Technology Preview release.
+- The `oc` command installed on your workstation.
+
+[NOTE]
```

Comment:
This seems pretty impactful. I'd recommend making it an IMPORTANT or a WARNING.

Reply:
Copy/pasted this from the other FeatureGate docs, where it's a NOTE. 🤷 But I agree, will bump it up.